<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorWidget</class>
 <widget class="QWidget" name="ColorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorWidget</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="paletteLabel">
        <property name="text">
         <string>Choose palette:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="paletteCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="paletteStack"/>
    </item>
   </layout>
  </widget>
  <widget class="GradientWidget" name="test_01" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="GradientSlider" name="test_02" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>21</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ctop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cmid">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cbottom">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GradientWidget</class>
   <extends>QWidget</extends>
   <header>GradientWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GradientSlider</class>
   <extends>QWidget</extends>
   <header>GradientSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
